rubygem-foreman_expire_hosts/foreman_expire_hosts.spec

```diff
@@ -0,0 +1,97 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_expire_hosts
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+
+Summary:    A Foreman plugin to allow hosts to expire.
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    2.0.0
+Release:    1%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        https://github.com/theforeman/foreman_expire_hosts
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Source1:    %{gem_name}.cron.d
+
+Requires:   foreman >= 1.10.0
+Requires: %{?scl_prefix_ruby}ruby(release)
```

Comment:
also require `%{?scl_prefix_ruby}rubygems` and fix inconsistent indentation, please